index.html

```diff
@@ -209,10 +237,32 @@ <h1 class="header-text">About
                 <hr>
             </div>
         </section>
+        <footer>
+            <section class="contacts">
+                <div class="contact-intro">
+                    <h1 class="contact-me">Contact me</h1>
+                    <p class="paragraph">
+                        If you have an application you are interested in developing, a
+                        feature that you need built or a project that needs coding. I’d love
+                        to help with it
+                    </p>
+                </div>
+                <form class="contact-section" action="#" method="post">
+                    <input id="name" class="text-holder1" type="text" name="name" maxlength="30" required
+                        placeholder="Name" />
+                    <input id="email" class="text-holder1" type="email" name="name" required placeholder="Email" />
+                    <textarea id="message" class="text-area" placeholder="Write your message here" maxlength="500"
+                        required></textarea>
+                    <div id="form-button">
+                        <button class="btn" type="submit" name="sumit">
+                            Get In Touch
+                        </button>
+                        <p id="form-error"></p>
+                    </div>
```

Comment:
- [ ] please make sure to use the right font-family, font-size, font-weight and text color in your inputs in order to match the Figma design 
- [ ] 
![image](https://user-images.githubusercontent.com/61790587/171682188-1f7ef186-fde4-4429-8e8b-d5b03caa38b6.png)